Patches/ExilePatch.cs

```diff
@@ -73,10 +73,10 @@ static void WrapUpPostfix(NetworkedPlayerInfo exiled)
         if (CLThingy && exiled != null)
         {
             // Reset player cam for exiled desync impostor
-            if (exiled.Object.HasDesyncRole())
-            {
-                exiled.Object?.ResetPlayerCam(1f);
-            }
+           //if (exiled.Object.HasDesyncRole())
+           // {
+           //     exiled.Object?.ResetPlayerCam(1f);
```

Comment:
Wait, if desync player will exiled, he not get black screen?

Reply:
oh wait maybe

Reply:
but the impostors are synced in exilepatch and that seems to work. So probably not needeed but this obviously needs testing.

I will sync exiled player and see

Reply:
looks like my idea works, so the desync exiled player, was actually not neccessary to leave a dead body afterwards